docs/reference/commandline/image_prune.md

```diff
@@ -62,6 +63,28 @@ deleted: sha256:2c675ee9ed53425e31a13e3390bf3f539bf8637000e4bcfbb85ee03ef4d910a1
 Total reclaimed space: 16.43 MB
 ```
 
+## Filtering
+
+The filtering flag (`-f` or `--filter`) format is of "key=value". If there is more
+than one filter, then pass multiple flags (e.g., `--filter "foo=bar" --filter "bif=baz"`)
+
+The currently supported filters are:
+
+* until (`<timestamp>`) - filter images created before given timestamp
```

Comment:
Perhaps, instead of "filter", describe what the result is; "only remove images created before given timestamp"